src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -121,6 +121,16 @@ static AbstractTriviaFormatter()
         /// </summary>
         protected abstract bool IsEndOfLine(SyntaxTrivia trivia);
 
+        /// <summary>
+        /// check whether given trivia is _ in VB or not
+        /// </summary>
+        protected abstract bool IsLineContinuation(SyntaxTrivia trivia);
+
+        /// <summary>
+        /// check whether given trivia is a Comment in VB or not
```

Comment:
why "in vb or not"?

Reply:
C# does not have a LineContunuation character so this allows returns false. I will update comment.

Reply:
@paul1956 Possibly `throw ExceptionUtilities.Unreachable;`, but I'm not very sure this is suitable here. I'd personally first try making most changes in the language-specific class. In case this was very hard, I'd go with the `throw` approach for C#.

Reply:
@Youssef1313 I think changing name if IsComment to IsVisualBasicComment and always return false for IsLineContinuation makes it clear and people will not be confused and try to use the IsComment in C# in the future.

Reply:
Can you take the approach Sam mentioned instead?